index.html

```diff
@@ -12,8 +12,149 @@
+            </div>
+            <div class="card-element__bottom">
+                <button class="card-button "><i class="far fa-heart"></i></button>
+                <button class="card-button "><i class="fas fa-share-alt"></i></button>
+                <button class="card-button "><i class="fas fa-ellipsis-v"></i></button>
+            </div>
+        </div>
+    </div>
 
-    
+    <h1>only text card</h1>
+    <div style="margin: 50px;">
+        <div class="card-vertical smooth-square-radius">
+            <div class="card-vertical-upper-text">
+                <h1 class="card-heading">Mustang Muscles</h1>
+                <p class="sub-text">this is power</p>
+                <div class="card-vertical-text">
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsam sequi quia magnam, ex animi explicabo
+                    consequu Lorem ipsum dolor sit amet consectetur adipisicing elit. Consequatur, aliquam? Maxime id,
+                    exercitationem expedita quasi praesentium explicabo laboriosam sit quia, possimus nostrum
+                    reprehenderit
+                    iure deleniti tempore accusantium odit aliquid unde?
+                </div>
+                <div class="card-element__bottom">
+                    <button class="card-button "><i class="far fa-thumbs-up"></i></button>
+                    <button class="card-button "><i class="fas fa-share"></i></button>
+                    <button class="card-button "><i class="far fa-bell"></i></button>
+                </div>
+            </div>
+        </div>
+
+        <h1>text-overlay- card</h1>
+
+        <div style="margin: 50px;">
+            <div class="card-vertical smooth-square-radius">
+                <div class="card-vertical-overlay-text">
+                    <h1 class="card-heading">Mustang Muscles</h1>
+                    <p class="sub-text">this is power</p>
+                </div>
+                <div class="card-vertical-text">
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsam sequi quia magnam, ex animi
+                    explicabo
+                    consequu Lorem ipsum dolor sit amet consectetur adipisicing elit. Consequatur, aliquam? Maxime
+                    id,
+                    exercitationem expedita quasi praesentium explicabo laboriosam sit quia, possimus nostrum
+                    reprehenderit
+                    iure deleniti tempore accusantium odit aliquid unde?
+                </div>
+                <div class="card-element__bottom">
+                    <button class="card-button "><i class="far fa-thumbs-up"></i></button>
+                    <button class="card-button "><i class="fas fa-share"></i></button>
+                    <button class="card-button "><i class="far fa-bell"></i></button>
+                </div>
+            </div>
+        </div>
+
+        <h2>card with dismiss and shadow</h2>
+        <div style="margin: 50px;">
```

Comment:
try to avoid inline styling

Reply:
Thanks for reviewing it buddy its just for now reviewing purpose will trash all html its just for clarity in peers review